app/loader.cpp

```diff
@@ -80,7 +85,7 @@ bool Loader::load( const QString &filePath )
   loop.exec();
 
   bool res = true;
-  if ( mProject->fileName() != filePath )
+  if ( mProject->fileName() != filePath || force )
```

Comment:
Isn't this condition always true?
```
if ( not the same || is the same)
```

Reply:
Maybe we would not need the `force` flag

Reply:
Force flag is not always true, only if a project has been synced and diff contains project file as Martin suggested. It has been added to extend and not change functionality of the original method and its used only after synced (not always though).

Reply:
I understand it is not always true, but the flag is basically being set to true when `mProject->homePath() == projectDir` and here is a comparison of something similiar, so if we moved the flag condition here, it would be: `mProject->fileName() != filePath || mProject->homePath() == projectDir` and that is what I thought is always true. But I sense that there is something I am missing.

Reply:
I think its mergeable. However, its good though that there is kind of extra step between `syncFinsihed` signal and `load` function, where local manager checks if synced project is currently active project. However, without that extra step, the condition will be always True as Tomas said, which we dont want to have. So the condition (extra step) must be there and cannot be omitted and send it directly to as a parameter to load function, otherwise any project after sync would be loaded.